### ISSUE TITLE
Module not found: Can't resolve 'create-react-class'

### ISSUE THREAD

**Issue by ggsjyoon** [kciter/react-barcode#16]
Hello. I started getting this error since I updated `redux-persist` from `4.8.0` to the latest version with Yarn 1.2.1.

```
./node_modules/react-barcode/lib/react-barcode.js
Module not found: Can't resolve 'create-react-class' in '..\node_modules\react-barcode\lib'
```

```
"dependencies": {
    "@types/history": "^4.5.0",
    "@types/jest": "^20.0.6",
    "@types/material-ui": "^0.18.2",
    "@types/moment-timezone": "^0.5.0",
    "@types/node": "^8.0.20",
    "@types/react": "^16.0.18",
    "@types/react-big-calendar": "^0.15.0",
    "@types/react-dnd": "^2.0.34",
    "@types/react-dnd-html5-backend": "^2.1.8",
    "@types/react-dom": "^16.0.2",
    "@types/react-redux": "^4.4.40",
    "@types/react-router-dom": "^4.0.4",
    "@types/react-router-redux": "^5.0.0",
    "@types/redux": "^3.6.0",
    "@types/redux-logger": "^3.0.0",
    "@types/redux-promise-middleware": "^0.0.6",
    "@types/validator": "6.3.0",
    "lodash": "^4.17.4",
    "material-ui": "0.19.2",
    "material-ui-datatables": "^0.18.2",
    "material-ui-fullscreen-dialog": "0.7.1",
    "moment-timezone": "^0.5.13",
    "react": "^16.0.0",
    "react-barcode": "^1.3.0",
    "react-big-calendar": "^0.16.1",
    "react-dnd": "^2.5.4",
    "react-dnd-html5-backend": "^2.5.4",
    "react-dock": "^0.2.4",
    "react-dom": "^16.0.0",
    "react-phone-number-input": "^0.12.10",
    "react-print": "^1.3.1",
    "react-redux": "5.0.6",
    "react-router-dom": "^4.1.1",
    "react-router-redux": "next",
    "react-tap-event-plugin": "^2.0.1",
    "react-transition-group": "^1.1.1",
    "redux": "3.7.2",
    "redux-logger": "^3.0.1",
    "redux-persist": "^5.2.2",
    "redux-promise-middleware": "4.2.1",
    "reselect": "^3.0.1",
    "validator": "8.2.0",
    "zxcvbn": "^4.4.2"
  },
  "devDependencies": {
    "react-scripts-ts": "^2.8.0"
  },
  "scripts": {
    "start": "react-scripts-ts start",
    "build": "react-scripts-ts build",
    "test": "react-scripts-ts test --env=jsdom",
    "eject": "react-scripts-ts eject"
  }
```

**Comment by ggsjyoon** [comment]
I installed `create-react-class` myself but ran into this runtime error. I'm not sure what's going on :worried: 

![error](https://user-images.githubusercontent.com/4350583/32417738-414774d0-c22c-11e7-8604-0bb81f4164c2.JPG)

**Comment by kciter** [comment]
Bug fixed.
Just update 1.3.1 :)

**Comment by ggsjyoon** [comment]
Module not found issue is resolved in `v1.3.1`, but it seems that `jsbarcode` options are not working proplery.

https://github.com/kciter/react-barcode/issues/17
